cpp/server/ct-server.cc

```diff
@@ -1,18 +1,11 @@
 /* -*- indent-tabs-mode: nil -*- */
 
-#include <chrono>
-#include <csignal>
-#include <cstring>
-#include <event2/buffer.h>
 #include <event2/thread.h>
-#include <functional>
 #include <gflags/gflags.h>
 #include <iostream>
-#include <memory>
-#include <mutex>
-#include <openssl/crypto.h>
 #include <openssl/err.h>
 #include <signal.h>
+#include <stdlib.h>  // for exit
```

Comment:
I'd skip the comment there, I'm pretty sure we use something else from there, or that we will, in the short future (so this comment will soon be or is already lying). :wink: